corehq/apps/accounting/management/commands/upgrade_software_plan_version.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+from django.core.management import BaseCommand
+
+from django_prbac.models import Role
+
+from corehq.apps.accounting.models import SoftwarePlanVersion
+
+logger = logging.getLogger(__name__)
+
+
+class OldRoleDoesNotExist(Exception):
+    pass
+
+
+class NewRoleDoesNotExist(Exception):
+    pass
+
+
+def upgrade_software_plan_version(old_role, new_role, limit_to_plans=None, dry_run=False):
+    """
+    :param old_role: slug for role to search for
+    :param new_role: slug for role that new software plan version should reference
+    :param limit_to_plans: optionally limit to specific plan names
+    :param dry_run: if False, will make changes to the DB
+    :return: a list of plan names that were upgraded
+    """
+    dry_run_tag = '[DRY_RUN]' if dry_run else ''
+
+    try:
+        Role.objects.get(slug=old_role)
```

Comment:
is this supposed to assign to an object? or is it just testing that it exists? if latter, a small comment might be helpful

Reply:
this would read much better imo